lighthouse-core/scripts/launch-chrome.sh

```diff
@@ -1,25 +1,44 @@
 #!/usr/bin/env bash
 
+# Running:
+#     npm run chrome
+# or
+#     npm explore -g lighthouse -- npm run chrome
+
+# You can also add custom cmd line flags for Chrome:
+#     npm run chrome -- --show-paint-rects
+
 CHROME_ARGS=$@
 
+URL_TO_LOAD="about:blank"
```

Comment:
maybe `INITIAL_URL` or something like that so it doesn't look like something you need to update to run lighthouse?